### PR TITLE
2914 map sidepane fix

### LINKED ISSUE #2914: Side pane does not appear
## Describe the bug
Side pane of the map does not appear if the node is already focused

## To Reproduce
1. Drag a node on the map
2. Click on the same node
3. Side pane does not expand
map_side_pane_bug

## Expected behavior 
Side pane should expand regardless of focused element

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do?

Fixes #2914

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by manual run, clicking, dragging, etc.
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -154,8 +154,8 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - A bug in network map page that caused delay of telemetry log loading. #1545
 - Windows "run as a user" powershell command for manual agent runs. #1556
-- A bug where T1216_random_executable.exe was copied to disk even if the signed
-  script proxy execution PBA was disabled. #1864
```
Comment (VakarisZ) [inline CHANGELOG.md:158]: Not related to PR, but pretty small change

**Review by ilija-lazoroski — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -154,8 +154,8 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
...
+- A bug in the map where side pane would not appear if the node was
+  dragged around before click. #2914
```
Comment (mssalvatore) [inline CHANGELOG.md:158]: Was this bug present in the last release? Or did we introduce it since then?

Comment (shreyamalviya) [reply CHANGELOG.md:158]: The line you changed was added in #2462.

Comment (shreyamalviya) [reply CHANGELOG.md:158]: After being removed in #2460 first. Never mind, it was most likely always there.

**Review by shreyamalviya — APPROVED** [review]

**Pull request merged**